src/js/view/view.js

```diff
@@ -340,9 +350,59 @@ function View(_api, _model) {
         playerViewModel.change('playlistItem', onPlaylistItem);
         // Triggering 'resize' resulting in player 'ready'
         _lastWidth = _lastHeight = null;
+
+        // Setup floating scroll handler
+        if (_floatingConfig && OS.mobile) {
+            _model.change('inDom', (model, val) => {
```

Comment:
instead of trying to catch every place where we're dealing with `inDOM` here, I used a catch-all handler in setup, that way the floating view doesn't care where in dom is changed, it can respond regardless

Reply:
Isn't the only thing you need to do is exit early in `throttledMobileFloatScrollHandler` if not inDOM?